code/modules/uplink/uplink_items.dm

```diff
@@ -449,7 +449,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
 			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
 	item = /obj/item/storage/box/syndie_kit/guardian
-	cost = 20 //SKYRAT EDIT CHANGE - ORIGINAL: 18 - Stupidly powerful considering traitors get 35tc now.
+	cost = 19 //SKYRAT EDIT CHANGE - ORIGINAL: 18 - Stupidly powerful considering traitors get 35tc now. You can offset the negatives with decent armor or an esword on top of that.
```

Comment:
having armor on doesn't mean anything for the user, the damage is transferred through the holoparasite 1:1 brute, bypassing armor.

Reply:
It's still a very powerful thing I've seen take down multiple sec players at once, even in 3+v1(2) confrontations. Combine this with something like an esword or a pistol and you have a stupidly powerful opponent. These things were 15tc on oldbase (out of 20) for a pretty good reason, and I'm leaving quite a lot of leftover change too.

If traitors can't make due with 16tc after buying a co op partner that can heal them, teleport them, or just super murder people, especially with sec guns being pretty weak at the moment, I'm gonna have to say it is completely on them.

Reply:
sec guns are still very strong, and sec has a huge numbers advantage. Heavy Roleplay works to security's advantage because it allows them to easily dogpile around any perceived threat while they're RPing

Reply:
Yeah. My above points touched on this. Holoparas suck  if the player is new or just bad/lagging. You have a unavoidably destructive liability tethered to you.